gofedlib/repository/gitlocalclient.py

```diff
@@ -5,13 +5,9 @@
 
 class GitLocalClient(object):
 
-	def __init__(self, repo_directory, repo_info = None):
```

Comment:
This is needed to pass additional data, like what repository is being processed.